frontend/src/pages/Marketplace.jsx

```diff
@@ -0,0 +1,214 @@
+// Standard React imports
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+
+// Styles
+import 'Styles/common/uclapi.scss';
+
+// Legacy
+import 'Styles/navbar.scss';
+
+// Images
+// Backgrounds
+import balloons from 'Images/home-page/balloons.jpg';
+import market from 'Images/marketplace/market.svg';
+import logo from 'Images/home-page/logo.svg';
+import arrow from 'Images/marketplace/arrow-left.svg';
+
+// Common Components
+import { Row, Column, TextView, ButtonView, CardView, ImageView, NavBar, Footer } from 'Layout/Items.jsx';
+
+// Grab titles and descriptions of app
+import { allApps } from 'Layout/data/app_pages.jsx';
+
+class Marketplace extends React.Component {

```

Comment:
Why are there 2 very similar components, one named Marketplace and the other AppPage? `<Marketplace>` doesn't seem to be used?

Reply:
Marketplace is the page that shows all of the apps /marketplace and apppage is an indivudal app page /marketplace/:appid: